packages/serverpod/lib/src/server/endpoint_dispatch.dart

```diff
@@ -72,12 +200,6 @@ abstract class EndpointDispatch {
       return ResultInvalidParams('Endpoint $path is not a valid endpoint name');
     }
 
-    // Find correct connector
-    var connector = getConnectorByName(path);
-    if (connector == null) {
-      return ResultInvalidParams('Endpoint $path does not exist');
-    }
-
     // Read query parameters
     var queryParameters = <String, dynamic>{};
```

Comment:
Suggestion: Move the `handleUriCall` method out from endpoint_dispatch.

This is now strictly transport layer code right?

Reply:
Should be! Will try to move it the `server`!

Reply:
[1961a38](https://github.com/serverpod/serverpod/pull/2676/commits/1961a38ff0ce7b4dfb098d2ba1fd80b5dbf20c99)